homeassistant/components/tractive/strings.json

```diff
@@ -30,15 +30,22 @@
       }
     },
     "sensor": {
+      "activity": {
+        "name": "Activity",
+        "state": {
+          "ok": "OK",
```

Comment:
Is that really the only two states it can have?

Reply:
There is no documentation for the API (reverse engineering) and for my two dogs these values are always `good` or `ok`.

Reply:
Right. We should implement `ENUM` device_class and use the options. Problem is of course if further options are available it will raise but I think we should do it anyway.

Reply:
Honestly I started this change with enum and removed it 😀 I’ll add it once again.